CorsixTH/Lua/rooms/research.lua

```diff
@@ -185,10 +185,7 @@ function ResearchRoom:commandEnteringPatient(patient)
         self.hospital.discover_autopsy_risk = self.hospital.discover_autopsy_risk + 10
       end
       if patient.hospital then
-        -- NB: Do not use patient:setHospital(nil) as that also causes despawning, which
-        -- has some bad effects if the autopsy machine is directly next to the room's door
         hosp:removePatient(patient)
-        patient.hospital = nil
```

Comment:
Do we know of a specific reason to set the hospital to nil, that is no longer there?